gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/nfp_node_driver.py

```diff
@@ -634,8 +635,6 @@ def _create_network_function(self, context):
                 raise VipNspNotSetonProvider()
 
             for provider_port in service_targets['provider_ports']:
```

Comment:
We dont need this for loop itself since all it does is allowed address pair port update